### ISSUE TITLE
[DribbblishDynamic] Bigger cover and playlist names?

### ISSUE THREAD

**Issue by tomini** [morpheusthewhite/spicetify-themes#202]
Hi!
How can I make the bigger cover art in DribbblishDynamic...well, even bigger :D I was successful with making it bigger, but it resulted in big gap by the right side and player progress bar. Perhaps some variable that multiplies pixel value of cover size to make gap in pixels between side and bottom progress bar.
Even better if I could make that bigger cover even bigger and somehow retain the cover art that is next to the name of song. It will vanish once I enable the big cover art version.

And the playlist names. I'm lost in all those playlists without their names. Tried to find it in user.css but was unsuccessful. Can I somehow enable playlist/album names for the left sidebar?

Thanks a lot!

EDIT: I found a percentage next to a song name that is currently playing. What does that mean? The amount of people that heart it from all who listened to it? Thanks again.

**Comment by Arecsu** [comment]
I would like to disable that percentage next to the song as well. I don't know what it means.

Also, would be great to re-design some part of the ui/ux to have the name of the playlists displayed. UX wise, it is a huge regression and makes navigating through playlists and finding the one that I want very difficult. It looks great, no doubt about it. But speaking about usability it is just worse.

Rest of the theme is beautiful. Just incredible. Discovered this after Spotify literally killed their UI with a new update and this was a god send.

**Comment by morpheusthewhite** [comment]
@JulienMaille

**Comment by JulienMaille** [comment]
@tomini can you show a mpckup of what you want to achieve? What do you use the "fullscreen" mode?

Regarding the percentage, I should mention it in the Readme, this is the track popularity according to Spotify.

**Comment by tomini** [comment]
> @tomini can you show a mpckup of what you want to achieve? What do you use the "fullscreen" mode?
> 
> Regarding the percentage, I should mention it in the Readme, this is the track popularity according to Spotify.

![image](https://user-images.githubusercontent.com/41676221/100360756-ab2ac080-2ff9-11eb-86a8-e28267e9a643.png)
I marked the changes with red rectangles. Bigger cover on the right (can be enabled with arrow on the small cover), but that disables the small cover next to name of the song, so I added it to the mockup...so 2 covers of currently playing music if possible. But most important would be the names of playlists on the left side.
And if possible - disable the percentage next to songs.

Thank you
